include/LieGroupControllers/impl/ControllerBase.h

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file ControllerBase.h
+ * @authors Giulio Romualdi
+ * @copyright This software may be modified and  distributed under the terms of
+ * the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef LIE_GROUP_CONTROLLERS_IMPL_CONTROLLER_BASE_H
+#define LIE_GROUP_CONTROLLERS_IMPL_CONTROLLER_BASE_H
+
+#include <LieGroupControllers/impl/traits.h>
+
+namespace LieGroupControllers
+{
+
+/**
+ * ControllerBase describes a base controller
+ * @tparam _Derived type of the Derived class. Necessary to implement the Curiously recurring
+ * template pattern.
+ */
+template <class _Derived> class ControllerBase
+{
+public:
+    using State = typename internal::traits<_Derived>::State; /**< State type */
+    using Vector = typename internal::traits<_Derived>::Vector; /**< Here we consider a Vector an
+                                                                   element of the tangent space of
+                                                                   the Group. */
```

Comment:
I am a bit confused by this comment here. How do you map in general the tangent space to a vector? Or you meant "tangent space at the identity" here?

Reply:
Yes I update the documentation

Reply:
This https://github.com/dic-iit/lie-group-controllers/pull/1/commits/afb9be36200e1667f411d012e6134d6d60f42ef6 fixes the documentation